src/content/getting-started/operations/autoscaling/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+linkTitle: Autoscaling
+title: Autoscaling
+description: Guides explaining all you need to know about cluster autoscaling (horizontal and vertical).
```

Comment:
```suggestion
description: Guides explaining all you need to know about cluster autoscaling.
```

I would leave this out of the description

Reply:
Do you have an option that's more than 70 char long?